alvr/dashboard/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 settings-schema = { git = "https://github.com/alvr-org/settings-schema-rs", rev = "676185f" }
 statrs = "0.16"
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls", "blocking", "json"] }
```

Comment:
Please use ureq instead of reqwest, it's already imported and it's sync instead of async (no tokio dependency).

Reply:
removed reqwest, using ureq